src/tiktok_uploader/upload.py

```diff
@@ -748,9 +749,10 @@ def _post_video(driver) -> None:
     logger.debug(green("Clicking the post button"))
 
     try:
+        tmp = "/html/body/div[1]/div/div[2]/div[2]/div/div/div/div/div[3]/div/div[2]/div[10]/button[1]"
         post = WebDriverWait(driver, config["implicit_wait"]).until(
             EC.element_to_be_clickable(
-                (By.XPATH, config["selectors"]["upload"]["post"])
+                (By.XPATH, tmp) # config["selectors"]["upload"]["post"] was instead of tmp
```

Comment:
You should change value in config file instead of tmp variable and use this value `//button[@data-e2e="post_video_button"]`

Reply:
@lavanphat  fixed